### ISSUE TITLE
Why is the initOAuth function commented?

### ISSUE THREAD

**Issue by rmarpozo** [springfox/springfox#788]
Why is the initOAuth function commented in swagger-ui.html?

```javascript
if (typeof initOAuth == "function") {
                        /*
                         initOAuth({
                         clientId: "your-client-id",
                         realm: "your-realms",
                         appName: "your-app-name"
                         });
                         */
                    }
```

The authenticate button never appears.

**Comment by dilipkrish** [comment]
Need to uncomment it and make those values substituable. Thanks for catching that!

**Comment by rmarpozo** [comment]
Ok, thanks!

**Comment by dilipkrish** [comment]
This should now be fixed in 2.0.2-SNAPSHOT
